packages/server/db/init-mongo.js

```diff
@@ -0,0 +1,15 @@
+print('Start #################################################################');
+
+cardzdb = db.getSiblingDB('cardzdb');
+
+cardzdb.createUser({
+    user: 'appadmin',
+    pwd: 'appadminpassword',
+    roles: ['readWrite']
+})
+
+cardzdb.createCollection('users');
+cardzdb.createCollection('games');
+cardzdb.createCollection('logs');
+
+print('END #################################################################');
```

Comment:
is this file automatically executed when starting mongo?

Reply:
Correct!
This is to initialise the container if it's the first time it's being built.